### PR TITLE
dao added

### PR DESCRIPTION
**Pull request by GSTharun**

### CODE REVIEW

File: src/main/java/com/ty/password/config/ConfigPass.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = "com.ty.password")
```
Comment (sagarb2722) [inline src/main/java/com/ty/password/config/ConfigPass.java]: give braces for base package

File: src/main/java/com/ty/password/config/ConfigPass.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+public class ConfigPass {
+	
+	@Bean("em")
+	@Scope("prototype")
+	public EntityManager getEntityManagerFactory()
+	{
+		return Persistence.createEntityManagerFactory("password").createEntityManager();
```
Comment (sagarb2722) [inline src/main/java/com/ty/password/config/ConfigPass.java:21]: once check that create entity manager then will continue the further process

File: src/main/java/com/ty/password/dao/Userdao.java [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
+		User user = entityManager.find(User.class, id);
+		EntityTransaction entityTransaction = entityManager.getTransaction();
+		entityTransaction.begin();
+		entityManager.remove(user);
+		entityTransaction.commit();
+
+		return true;
+	}
+	
+	public List<User> findUser(){
```
Comment (sahanaagowda) [inline src/main/java/com/ty/password/dao/Userdao.java]: give proper method names

File: src/main/java/com/ty/password/config/ConfigPass.java (hunk 1 above)
Comment (GSTharun) [reply src/main/java/com/ty/password/config/ConfigPass.java]: okayy bro

Comment (vvviin) [inline src/main/java/com/ty/password/config/ConfigPass.java]: add bracket for basepackages

**Review by vvviin — APPROVED** [review]

**Review by sagarb2722 — APPROVED** [review]

**Review by sahanaagowda — APPROVED** [review]

**Pull request merged**